HW_3/app_task3.py

```diff
@@ -0,0 +1,39 @@
+'''Задание №8
+Создать форму для регистрации пользователей на сайте.
+Форма должна содержать поля "Имя", "Фамилия", "Email",
+"Пароль" и кнопку "Зарегистрироваться".
+При отправке формы данные должны сохраняться в базе
+данных, а пароль должен быть зашифрован.
+'''
+
+from flask import Flask,render_template,request, redirect, url_for
+from hashlib import sha256
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf.csrf import CSRFProtect
+from HW_3.forms import Registration
+from HW_3.model import User,db
```

Comment:
У вас папка HW_3 не является модулем, так как отсутствует файл __init__.py
```suggestion
from forms import Registration
from model import User,db
```